### DISCUSSION TITLE
How can I convert all these files to make it work with 3dmigoto?

### DISCUSSION THREAD

**Discussion by Veois** [zeroruka/GI-Assets#240] [Q&A]
Hi! How can I convert all these .png files to .dds and create .ini files to make it work with 3dmigoto? I just haven't found instructions on this anywhere... Or is it done in some other way? I saw a tutorial on how to get a model from kenshin to change it, and there are a lot of them, it turns out that I have to change each texture myself? Or is there some kind of auto-compiler? I'm sorry in advance if the questions are stupid, I just don't understand this topic very much...

**Comment by zeroruka** [comment]
These files are raw models extracted from game assets, if you want to make mods with 3dmigoto, you have to use files in this repo instead: [GIMI Assets](https://github.com/SilentNightSound/GI-Model-Importer-Assets)
